bindings.cpp

```diff
@@ -1814,7 +1839,13 @@ bool DeRestPluginPrivate::checkSensorBindingsForAttributeReporting(Sensor *senso
         sensor->modelId().startsWith(QLatin1String("SN10ZW")) ||
         sensor->modelId().startsWith(QLatin1String("SF2")) ||
         // Netvox
-        sensor->modelId().startsWith(QLatin1String("Z809A")))
+        sensor->modelId().startsWith(QLatin1String("Z809A")) ||
+        // Samsung SmartPlug 2019
+        sensor->modelId().startsWith(QLatin1String("ZB-ONOFFPlug-D0005")))
+        // Aurora
+        sensor->modelId().startsWith(QLatin1String("DoubleSocket50AU")))
+        // Bosch
+        sensor->modelId().startsWith(QLatin1String("ISW-ZPR1-WP13")))
```

Comment:
This shouldn't compile? Missing some || ?

Reply:
D'oh. Fixed, sorry for that one.